docs/configuration/git-providers/bitbucket.md

```diff
@@ -0,0 +1,35 @@
+# Bitbucket
+
+Since Bitbucket uses OAuth, the first step is to register a consumer for Codacy on Bitbucket side.
+
+## Create a consumer
+OAuth needs a key and secret, together these are known as an OAuth consumer. You can create a consumer on any existing individual or team account. To create a consumer, do the following:
+
+  1. From your avatar in the bottom left, click Bitbucket settings.
+  1. Click *"OAuth"* from the left navigation and click the *"Add consumer"* button.  
+     1. Set the name of the consumer (e.g. *"Codacy"*).
+     1. Set the callback URL as *"http://<codacy_url>:<codacy_port>/login/Bitbucket?codacy_skip_ga=1"*.
+     1. Tick the *"This is a private consumer"* checkbox:
+        ![Bitbucket Consumer Configuration](./images/bitbucket-consumer-configuration.png)
+     1. Add the following permissions: 
+     ![Bitbucket Consumer Permissions](./images/bitbucket-consumer-permissions.png)
+     1. Click Save. 
+        The system generates a key and a secret for you.
+     1. Toggle the consumer name to see the generated Key and Secret value for your consumer.
+     ![Bitbucket Consumer Key and Secret](./images/bitbucket-consumer-key-and-secret.png)
+
+## Enable Bitbucket and set the key and the secret
+Set your configuration values for Bitbucket on the `values.yaml` file.
```

Comment:
This should be the `values-production.yaml` file instead:

```suggestion
Set your configuration values for Bitbucket on the `values-production.yaml` file.
```

Reply:
notice that we have `values.yaml` in every place 😨

Reply:
I know, I'm already fixing this in another PR for the Monitoring configuration and will also do another full review when both the Inquisition and Jonah's squads finish their current work on the Git providers. So don't worry about the other occurrences. :+1: